vuepress/docs/next/docs/reference/keycloak-cr.md

```diff
@@ -17,11 +17,11 @@ spec:
   imageName: "entando/entando-keycloak"
   dbms: "postgresql"
   ingressHostName: "test-keycloak.192.168.1.1.nip.io"
-  entandoImageVersion: "7.0.0"
```

Comment:
Does this entandoImageVersion still belong?

Reply:
Looking in the current files of this type, it seems that it has been removed